internal/client/server.go

```diff
@@ -93,7 +94,7 @@ func (c *Project) initLocalServer(ctx context.Context) (*grpc.ClientConn, error)
 		Timeout: 1 * time.Second,
 	})
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed opening boltdb path %s. Is another server already running against this db?: %w", path, err)
```

Comment:
This is mostly unrelated, but I spent a good long while trying to get to the bottom of why this change was breaking waypoint's local server mode. It turns out I wasn't initializing the plugin command with the `WithNoAutoServer()` option, so the plugin was trying to spool up another server, and crashing when it couldn't open the bolt's `data.db` file. The only thing I had to go on from the logs was this:

```
! Unrecognized remote plugin message: ! failed to create client: timeout
```

So i'm enhancing that log message here.